packages/core/cms/faststore/content-types.json

```
@@ -99,8 +99,27 @@
   {
     "id": "pdp",
     "name": "Product Page",
-    "isSingleton": true,
-    "configurationSchemaSets": []
+    "configurationSchemaSets": [
+      {
+        "name": "Settings",
+        "configurations": [
+          {
+            "name": "template",
+            "schema": {
+              "title": "Template",
+              "type": "object",
+              "properties": {
+                "value": {
+                  "title": "PDP template value (e.g. Slug: /apple-magic-mouse/p OR /department/category/subcategory/)",
+                  "type": "string",
+                  "description": "The value can be: the PDP slug template (e.g. /apple-magic-mouse/p). A /department/category/subcategory/ template (e.g. /office/ OR /office/chairs/ OR /office/chairs/classic). If this field is empty this template will be the generic PDP."
```

What about...?
```suggestion
                  "description": "Possible values: the PDP slug template (e.g. '/apple-magic-mouse/p'); A '/department/category/subcategory/' template (e.g. '/office/' OR '/office/chairs/' OR '/office/chairs/classic/'). If empty, this template will be the generic PDP."
```

---

Maybe follow the same pattern? Assuming that the user knows the page definitions
```suggestion
                  "description": "Possible values: the PDP slug template (e.g. '/apple-magic-mouse/p'); A PLP template (e.g. '/office/' OR '/office/chairs/' OR '/office/chairs/classic/'). If empty, this template will be the generic PDP."
```

---

I like mantis suggestion to keep the pattern, but I also think an example would be good! `office/chairs/classic`

---

Thank you, folks!

Let's keep some real examples in the docs, then.
- https://github.com/vtex/faststore/pull/2236